pro/src/components/pages/Offerers/Offerer/VenueV1/VenueCreation/__specs__/helpers.js

```diff
@@ -0,0 +1,126 @@
+import { screen } from '@testing-library/dom'
+import { waitFor } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+
+import fetchAddressData from 'components/pages/Offerers/Offerer/VenueV1/fields/LocationFields/utils/fetchAddressData'
+
+export const fieldLabels = {
+  name: { label: 'Nom du lieu', exact: false, type: 'text' },
+  bookingEmail: { label: 'Mail : *', exact: false, type: 'text' },
+  comment: { label: 'Commentaire', exact: false, type: 'text' },
+  address: { label: 'Numéro et voie', exact: false, type: 'text' },
+  city: { label: 'Ville', exact: false, type: 'text' },
+  postalCode: { label: 'Code postal', exact: false, type: 'text' },
+  longitude: { label: 'Longitude', exact: false, type: 'text' },
+  latitude: { label: 'Latitude', exact: false, type: 'text' },
+  type: { label: 'Type de lieu', exact: false, type: 'select' },
+  noDisabilityCompliant: {
+    label: 'Non accessibl',
```

Comment:
```suggestion
    label: 'Non accessible',
```

On est dans des tests donc ce n'est pas une fatalité, mais c'est toujours mieux :)